### PR TITLE
Celery service setup

### PR DESCRIPTION
**Pull request by sc15zs**
Add Celery service to docker compose file for running background tasks. 

Ticket: https://dluhcdigital.atlassian.net/browse/FPASF-102

### CODE REVIEW

File: docker-compose.yml
Comment (samuelhwilliams) [inline docker-compose.yml:8]: Weird, I thought Marc had already added this 🤔

**Review by samuelhwilliams — APPROVED** [review]
Looks good to go once the data-store PR is merged 👍

File: docker-compose.yml
Comment (gidsg) [inline docker-compose.yml:50]: Does the worker actually need these env vars? Does it start up without them? Would be good to get the env vars down to a minimal set.

Comment (sc15zs) [reply docker-compose.yml]: Removed these from celery setup.

**Pull request merged**
